src/w3c/headers.js

```diff
@@ -531,17 +534,17 @@ export function run(conf) {
   }
   if (Array.isArray(conf.wg)) {
     conf.multipleWGs = conf.wg.length > 1;
-    conf.wgHTML = joinAnd(conf.wg, (wg, idx) => {
-      return `the <a href='${conf.wgURI[idx]}'>${wg}</a>`;
+    conf.wgHTML = htmlJoinAnd(conf.wg, (wg, idx) => {
+      return hyperHTML`the <a href='${conf.wgURI[idx]}'>${wg}</a>`;
     });
     const pats = [];
     for (let i = 0, n = conf.wg.length; i < n; i++) {
-      pats.push(
+      const wgPatentURI =
         `a <a href='${conf.wgPatentURI[i]}' rel='disclosure'>` +
-          `public list of any patent disclosures  (${conf.wg[i]})</a>`
-      );
+        `public list of any patent disclosures  (${conf.wg[i]})</a>`;
+      pats.push(hyperHTML`${wgPatentURI}`);
```

Comment:
`hyperHTML` must be used for template strings when interpolating, not after. Here we have to merge the two lines into a single template string with hyperHTML so that every input can be auto escaped by hyperHTML.